src/ResourceManager/IotHub/Commands.IotHub/Commands.IotHub.csproj

```diff
@@ -37,30 +37,24 @@
     <Reference Include="Hyak.Common">
       <HintPath>..\..\..\packages\Hyak.Common.1.0.3\lib\portable-net403+win+wpa81\Hyak.Common.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.EventHub, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Why do you add this depednency on EventHub?  This creates a cross-depednency on EventHub managemrnt client between this module and the Evnethub module.  Please remove this, or consider moving any necessary methods into common code.